web/src/features/map-controls/MapControls.tsx

```diff
@@ -109,7 +109,7 @@ function DesktopMapControls() {
   };
 
   return (
-    <div className="pointer-events-none absolute right-3 top-2 z-30 hidden flex-col items-end md:flex">
```

Comment:
This change will mean the buttons are hidden behind the legend if you have the screen in a ridiculous size, unlikely case but think it's good to leave it at 30